scripts/base/protocols/ssl/files.zeek

```diff
@@ -6,27 +6,37 @@
 module SSL;
 
 export {
+	## Set this to false to remove the server certificate subject and
+	## issuer from the SSL log file. This information is still available
+	## in x509.log.
+	const log_include_server_certificate_subject_issuer = T &redef;
+
+	## Set this to true to include the client certificate subject
+	## and issuer in the SSL logfile. This information is rarely present
+	## and probably only interesting in very specific circumstances
+	const log_include_client_certificate_subject_issuer = F &redef;
```

Comment:
True. Doing these as redefs was partially me being lazy - since making them options will add a bunch more complexity.

I am obviously happy to do that - with adding tests that will probably take me another day or so. (I am actually not sure that we have any tests that check what happens if you change the columns of a log-file while Zeek is running - guess this is the time to add that).

Reply:
I tested this - runtime changes of log columns are currently not possible. I opened a bug in #1647. We can change this here once this bug has been fixed, before there does not seem to be much of a point.

I checked; this PR does not introduce any other new consts; everything else either was already existing or is an option.